docs/sphinx/source/whatsnew/v0.11.1.rst

```diff
@@ -13,6 +13,9 @@ Enhancements
 * Add new losses function that accounts for non-uniform irradiance on bifacial
   modules, :py:func:`pvlib.bifacial.power_mismatch_deline`.
   (:issue:`2045`, :pull:`2046`)
+* Add new parameters for min/max absolute air mass to
+  :py:func:`pvlib.spectrum.spectral_factor_firstsolar`.
```

Comment:
Is this whatsnew note sufficient or should I add something about the docs/warnings as well? I think that this (new parameters) is the main change though --- a new feature.
Perhaps changes to the warnings could be classed as (noteworthy?) changes to default behaviour(?)

Reply:
I think the note is sufficient.